docs/tutorials/01_electronic_structure.ipynb

```diff
@@ -108,159 +111,478 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For further information about the drivers see https://qiskit.org/documentation/nature/apidocs/qiskit_nature.second_q.drivers.html"
+    "Running this driver, will yield an `ElectronicStructureProblem`, Qiskit Nature's representation of the electronic structure problem which we are interested in solving. For further information about the drivers see https://qiskit.org/documentation/nature/apidocs/qiskit_nature.second_q.drivers.html"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<qiskit_nature.second_q.problems.electronic_structure_problem.ElectronicStructureProblem object at 0x15287045b700>\n"
```

Comment:
Having got this far, though print(problem) seems to produce nothing in the output (well a blank line) its just the basic instance print like this - I guess we have no str defined for it

Reply:
See similar previous threads. I'll resolve this one